docs/src/operate/registration/http_api.md

```diff
@@ -83,158 +83,79 @@ GET /v1/entities/{topic-id}
 curl http://localhost:8000/te/v1/entities/device/child01
 ```
 
-## Query entities
+## Update entity
+
+An existing entity can be updated using the PATCH API.
+But the updates are limited to the `@parent` and `@health` endpoints only.
+Other properties like `@type` and `@id` can not be updated after the registration.
+
+The `@parent` must be another device, as services cannot have children.
+Similarly, the `@health` endpoint must be a service, as only services publish their health statuses.
+The entities specified in either fields must be registered up-front, before they can be used.
```

Comment:
It would be more helpful to give common uses of the api rather then going into the "limitations" of the api. e.g.

```
The update api is typically used to:
* change the parent of a child device
* change the health topic of a service
```

Reply:
Resolved by 4ff49e7